src/components/radios/template.njk

```diff
@@ -18,6 +18,11 @@
   {% endif %}
 {% endfor %}
 
+{% set userRefinedClasses = ' ' + params.classes if params.classes %}
```

Comment:
From a user's perspective if they're using standard HTML this feels a bit weird.

1. start off with block level inputs
2. then want them inline so they add an inline modifier
3. then they run into this weird rendering so add additional modifier which essentially just undos 2.?

Reply:
if they set inline and get broken styling they might
a) clear floats in css directly 
b) add modifier and add styles
c) just use block

Not sure what we can do for them.